arm-iothub/2017-01-19/swagger/iothub.json

```diff
@@ -964,9 +964,452 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/certificates": {
+      "get": {
+        "tags": [ "GET" ],
+        "summary": "Get the certificate list.",
+        "description": "Get the certificate list.",
+        "operationId": "Certificates_Get",
```

Comment:
There two groups of methods `Certificates_*` and `Certificate_*`. As far as I understood, `Certificates` will have only one method `Get`. I would consider something like `IotHubResource_ListCertificates".